README.md

```diff
@@ -62,8 +62,8 @@ If you're curious to know more, see the discussion within https://github.com/ter
 - `project` - Name of project this VPC is meant to house (default: `Unknown`)
 - `environment` - Name of environment this VPC is targeting (default: `Unknown`)
 - `allocated_storage` - Storage allocated to database instance (default: `32`)
-- `engine_version` - Database engine version (default: `9.4.4`)
-- `instance_type` - Instance type for database instance (default: `db.t2.micro`)
+- `engine_version` - Database engine version (default: `11.5`)
```

Comment:
Since the upgrade to Terraform 0.12 is going to lead to a major version bump, I figured the default engine version should get bumped as well.